docs/api/rpc-experimental.md

```diff
@@ -0,0 +1,305 @@
+---
+id: rpc-experimental
+title: Experimental Methods
+sidebar_label: Experimental Methods
+---
+
+<blockquote class="warning">
+<strong>heads up</strong><br><br>
+
+This page refers to a **technical preview**.
+
+These features are not currently available on TestNet. They will be available in an upcoming release.
+
+</blockquote>
+
+Two new methods will be exposed:
+
+### `EXPERIMENTAL_genesis_config`
```

Comment:
We should align the style of the headers with the "RPC" document, where we use human-readable titles and pseudo-calls like `EXPERIMENTAL_genesis_config()`, `EXPERIMENTAL_genesis_records(pagination: Pagination)`

Reply:
✅